features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml

```diff
@@ -23,6 +23,19 @@
     <property name="eventReader" ref="eventReader"/>
   </bean>
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
+  
+  <bean id="trapReceiverSnmp4jImpl" class="org.opennms.netmgt.trapd.TrapReceiverSnmp4jImpl">
```

Comment:
There is already a TrapReceiverSnmp4jImpl bean in the file. You need to combine your new changes with it.